src/main/java/com/zendesk/maxwell/producer/AbstractProducer.java

```diff
@@ -1,18 +1,33 @@
 package com.zendesk.maxwell.producer;
 
-import com.codahale.metrics.Meter;
+import com.codahale.metrics.*;
```

Comment:
Can we import specific classes?

Reply:
I thought the example of the import in [AbstractAsyncProducer](https://github.com/zendesk/maxwell/blob/master/src/main/java/com/zendesk/maxwell/producer/AbstractAsyncProducer.java#L3) would be the style to match. Change both to specific imports?

Reply:
that'll be great!

Reply:
done d784d9a3